lib/index.js

```diff
@@ -51,15 +51,20 @@ const deleteFood = (event) => {
   .catch(error => console.log({error}));
 }
 
-const editFood = () => {
-  var newName = document.activeElement
-//method in process
-}
+function editFood()  {
+  var foodId = $(this).data('food-id').toString();
+  var html = $(this).html();
+  var updatedName = (this.getElementsByClassName('name')[0]).innerText;
```

Comment:
Remember if you are using jQuery above try to stick with that within the same file. I believe you can use grab the class from `this` after the fact i.e. `$(this).('name')`...